solidity/computation-datasource/paypal/PaypalExample.sol

```diff
@@ -1,27 +1,27 @@
-pragma solidity ^0.5.0;
+pragma solidity >= 0.5.0 < 0.6.0;
 
 import "github.com/oraclize/ethereum-api/oraclizeAPI.sol";
 
 contract PaypalExample is usingOraclize {
 
     event LogPaymentRe(string payment_info);
```

Comment:
Change to:
`event LogPaymentResult(string payment_info);`

Reply:
Well spotted. Wonder how long that's been like that‽

Reply:
Update: 4 months! And it was me who did it...oopsie!

Reply:
Fixed.